server/Arcadia.Assistant.Calendar/SickLeave/SendEmailSickLeaveActor.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace Arcadia.Assistant.Calendar.SickLeave
+{
+    using System;
+    using Akka.Actor;
+    using Akka.DI.Core;
+    using MimeKit;
+    using Configuration.Configuration;
+    using MailKit.Net.Smtp;
+    using MailKit.Security;
+    using Events;
+
+    public class SendEmailSickLeaveActor : UntypedActor
+    {
+        private readonly IEmailSettings mailConfig;
+        private readonly ISmtpSettings smtpConfig;
+        public SendEmailSickLeaveActor(IEmailSettings mailConfig, ISmtpSettings smtpConfig)
+        {
+            this.mailConfig = mailConfig;
+            this.smtpConfig = smtpConfig;
+        }
+
+        public static Props GetProps() => Context.DI().Props<SendEmailSickLeaveActor>();
+
+        protected override void OnReceive(object message)
+        {
+            switch (message)
+            {
+                case SickLeaveIsApproved ev:
+                    SendEmail(ev);
+                    break;
+            }
+        }
+
+        private void SendEmail(SickLeaveIsApproved Event)
+        {
+            using (var client = new SmtpClient())
+            {
+                var message = CreateMimeMessage(String.Format(mailConfig.Body, Event.UserId, Event.TimeStamp.ToString()));
```

Comment:
add some logging here. Like, 'Sending a sick leave email notification for user {0}'. And one for completion, so it'll help us to see if the message were sent